assets/src/amp-validation/amp-validated-urls-index.js

```diff
@@ -6,6 +6,6 @@ import domReady from '@wordpress/dom-ready';
 /**
  * Internal dependencies
  */
-import setValidationErrorRowsSeenClass from './set-validation-error-rows-seen-class';
+import setValidationErrorRowsClasses from './set-validation-error-rows-classes';
 
-domReady( setValidationErrorRowsSeenClass );
+domReady( setValidationErrorRowsClasses );
```

Comment:
It seems this JS file is no longer needed, as you can't toggle the reviewed or error status from the Validated URLs index page.

Reply:
Good catch! The entire JS file is removed in d92ab08fa9d8a84945939164811ba1b3aade1d1d.

Reply:
@pierlon Is this not actually needed per my last screenshot in https://github.com/ampproject/amp-wp/pull/6067#issuecomment-821633507?

Reply:
Yes the function is still being called via the enqueued `amp-validation-detail-toggle.js`:
https://github.com/ampproject/amp-wp/blob/23f423363afc524c1d58fb23bf75d53b8b01e330/assets/src/amp-validation/amp-validation-detail-toggle.js#L84